### PR TITLE
{numlib}[foss/2020a,intel/2020a] SuiteSparse v5.7.1

### PR DESCRIPTION
**Pull request by MaximeMoge**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.7.1-intel-2020a-METIS-5.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+versionsuffix = '-METIS-%s' % local_metis_ver
+
+homepage = 'https://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['https://github.com/DrTimothyAldenDavis/SuiteSparse/archive']
+sources = ['v%(version)s.tar.gz']
```
Comment (lexming) [inline easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.7.1-intel-2020a-METIS-5.1.0.eb]: Please add missing checksums
```suggestion
sources = ['v%(version)s.tar.gz']
checksums = ['5ba5add1663d51a1b6fb128b50fe869b497f3096765ff7f8212f0ede044b9557']
```

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 1 out of 3 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f852141feb478a5eb25a794e9b0f7ca4 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/a03a19b38939a21e30bcc08506d99b2a for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/da9453604a65cdbf6505e35df51176df for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/03316f9d9252a727995387c70636599b for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c54f696614071000804bb4d252d57660 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e7acaafe39d46f911575371ad81744cc for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/dd788af802b750ef240457ccdc604173 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @MaximeMoge!

**Pull request merged**